examples/tools/scene_viewer/scene_viewer_plugin.rs

```diff
@@ -4,13 +4,12 @@
 //! - Insert an initialized `SceneHandle` resource into your App's `AssetServer`.
 
 use bevy::{
-    gltf::Gltf, input::common_conditions::input_just_pressed, prelude::*, scene::InstanceId,
+    camera_controllers::free_cam::FreeCamController, gltf::Gltf,
```

Comment:
Would be cool to re-export this from top level crate and probably shorten it to just `FreeCam` maybe

Reply:
I think we overuse re-exports! Shortening it would be good though.